src/modules/Radio/RadioGroup.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { WithStylesProps, ThemeStyleSheetFactory } from "../../types";
+import { withStyles } from "../../base";
+
+interface IProps {
```

Comment:
extend StardardProps<"input">(?) just to keep the default props like children, styles, classname , onchange, disabled, value

Reply:
bump